app/models/story.rb

```diff
@@ -408,6 +408,7 @@ def check_tags
       return
     end
 
+    Prosopite.pause
```

Comment:
skipped because `taggings` is not created yet and cannot join `tag` to them

Reply:
I spent a few minutes tinkering with this, the `resume` can move up to line 425, the section `if taggings.reject...` isn't tripping it.